### DISCUSSION TITLE
IMPORTING DATE FROM CSV AND NUMBER OF ITEMS IN COLUMNS

### DISCUSSION THREAD

**Discussion by VADDIJASWANT** [kanriapp/kanri#721] [Ideas]
Hi everyone, first things first, this is a great tool! Thank you for putting in all the effort to build it. I have two feature requests. The first is the ability to import from CSV files. Typically, in our office, we receive bugs or work items in Excel or as regular points directly, so when I want to add them to the columns, I have to do so one by one, which takes a considerable amount of time. If there were a feature that allows for CSV import, it would be fantastic. Additionally, could you optionally display the number of items in each column? This would provide a quick overview of how many items are present in a particular column.

**Comment by trobonox** [comment]
Thanks for your feedback and I'm glad you're enjoying the app.

The CSV import is a bit tricky because there's no direct way to map a list of items to actual columns unless there is a column/category property. In that case, it might be very similar to the feature to import from a GitHub Project, which exports as a .tsv file (tab-separated values, essentially a CSV but with tabs as separators).

The feature to show the number of items in each column is actually already a thing, just go into the settings menu and under board preferences you should find the option "Display number of cards in column". If that's not what you are looking for, please let me know what you would expect for something like this.

**Reply by VADDIJASWANT** [reply]
Hi, the feature for the number of items in the column that I wanted was already there. Sorry, I didn't see that earlier. Regarding the import from TSV, I didn't know what TSV was until now. I will look into it and see if I can use it. Thanks a lot!

**Reply by trobonox** [reply]
Just for some extra context, the top-most line needs to have following fields so it can be imported:
```
Title	URL	Assignees	Status	Labels
```
Then below this first line, you just have one line representing one card with the values separated with tabs, with "status" being the column each task is located in.
